test/libyul/yulOptimizerTests/fullSuite/abi_example1.yul

```diff
@@ -473,7 +473,7 @@
 //         let i := _1
 //         for { } lt(i, length) { i := add(i, 1) }
 //         {
-//             abi_encode_t_array$_t_contract$_C_$55_$3_memory_to_t_array$_t_address_$3_memory_ptr(mload(srcPtr), pos)
+//             abi_encode_t_array$_t_contract$_C_memory(mload(srcPtr), pos)
```

Comment:
The rule for `t_contract$_C_` should replace it by `C`. Doesn't seem to be working.

Reply:
I think the issue is that the other replacement was already done. I'll see what I can do.